i18n/en.json

```diff
@@ -84,5 +84,9 @@
     "usage-consent-header": "Send usage reports",
     "usage-consent-explanation": "Help make the app better by letting us know how you use it. Data collected is anonymous. Read privacy policy for more details.",
     "usage-consent-yes": "Yes",
-    "usage-consent-no": "No"
+    "usage-consent-no": "No",
+    "softkey-yes": "Yes",
```

Comment:
The `yes`/`no` here should be the same keys as in the previous PR.

Reply:
Yea we could reuse the same labels here from the previous PR, which are `usage-consent-no` and `usage-consent-yes`. I intentionally created `softkey-yes` and `softkey-no` becuase these new labels are actually used with softkeys only, whereas `usage-consent-no` and `usage-consent-yes` are used in the radio list view. It adds a bit of redundancy and yet brings in some separation of concerns. But happy to update this if that doesn't really matter, please confirm